frontend/src/components/ScheduledEventsRunHistoryTableRow/ScheduledEventsRunHistoryTableRow.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import moment from 'moment';
+import React, { FunctionComponent } from 'react';
+import { convertStatus, getStatusClasses } from './utils';
+
+interface ScheduledEventsRunHistoryTableRowProps {
+  status: string;
+  started: string;
+  ended: string;
+}
+export const ScheduledEventsRunHistoryTableRow: FunctionComponent<ScheduledEventsRunHistoryTableRowProps> = (
+  props,
+) => {
+  const badgeClass = getStatusClasses(props.status);
+
+  return (
+    <tr>
+      <td>
+        <span className={classNames('badge', badgeClass)}>{convertStatus(props.status)}</span>
```

Comment:
Surprised you converted this ... I remember u left the original scheduled events table with its `wks` `mnths` stuff ... not complaining, I prefer this.